Tasks/DeleteFilesV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,12 +1,14 @@
 {
   "loc.friendlyName": "Delete files",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=722333)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=722333)",
```

Comment:
This file was auto updated on build, you should be able to ignore changes here. This just reflects past string changes that should have already been updated here.